redaxo/src/addons/install/pages/packages.add.php

```diff
@@ -141,21 +176,25 @@
                 <th class="rex-table-icon"><a class="rex-link-expanded" href="' . rex_url::currentBackendPage(['func' => 'reload']) . '" title="' . $package->i18n('reload') . '"><i class="rex-icon rex-icon-refresh"></i></a></th>
                 <th class="rex-table-min-width-4 rex-table-sort"><a class="rex-link-expanded" href="'.rex_url::currentBackendPage().'" title="'.$package->i18n('sort_default').'">' . $package->i18n('key') . '</a></th>
                 <th class="rex-table-min-width-4">' . $package->i18n('name') . ' / ' . $package->i18n('author') . '</th>
-                <th class="rex-table-min-width-4 rex-table-sort"><a class="rex-link-expanded" href="'.rex_url::currentBackendPage(['sort' => $sortNext]).'" title="' . $package->i18n('sort') . '"><span class="text-nowrap">' . $package->i18n('published_on') . '</span>&nbsp;<span><i class="rex-icon rex-icon-sort fa-sort'.$sortClass.'"></i></span></a></th>
+                <th class="rex-table-min-width-4 rex-table-sort"><a class="rex-link-expanded" href="'.rex_url::currentBackendPage(['sort' => $sortNextPublishedDate, 'sort_type' => 'published']).'" title="' . $package->i18n('sort') . '"><span class="text-nowrap">' . $package->i18n('published_on') . '</span>&nbsp;<span><i class="rex-icon rex-icon-sort fa-sort'.$sortByPublishedDateClass.'"></i></span></a></th>
+                <th class="rex-table-min-width-4 rex-table-sort"><a class="rex-link-expanded" href="'.rex_url::currentBackendPage(['sort' => $sortNextDownloads, 'sort_type' => 'downloads']).'" title="' . $package->i18n('sort') . '"><span class="text-nowrap">' . $package->i18n('downloads') . '</span>&nbsp;<span><i class="rex-icon rex-icon-sort fa-sort'.$sortByDownloadsClass.'"></i></span></a></th>
                 <th>' . $package->i18n('shortdescription') . '</th>
                 <th class="rex-table-action">' . $package->i18n('header_function') . '</th>
             </tr>
          </thead>
          <tbody>';
 
     foreach ($addons as $key => $addon) {
+        $downloads = array_sum(array_column($addon['files'], 'counter')) ?: 0;
```

Comment:
Die Gesamtzahl müsste auch direkt in der API verfügbar sein, auf Addon-Ebene.

Reply:
Die habe ich total übersehen, danke für den Hinweis. Das passe ich an. Bei Psalm habe ich ein paar Stellen gefunden, an denen ich die Kommentare verändern muss, aber dann muss ich die Baseline vermutlich neu erstellen.